plugins/cluster-status-backend/src/statusCheck/StatusCheck.ts

```diff
@@ -88,6 +88,14 @@ export class StatusCheck {
     }
   }
 
+  private convertToCpus = (cpus: string): string => {
+    if (cpus.slice(-1) === 'm') {
+      const cpusInMilicores = parseInt(cpus.slice(0, cpus.length - 1), 10)
+      return `${Math.round(cpusInMilicores / 1000)}`;
```

Comment:
```suggestion
      return Math.round(cpusInMilicores / 1000).toString();
```

Reply:
I think it makes more sense also, gonna change it to numbers for pods and cpus.